tests/plugins/test_accounts.py

```diff
@@ -252,7 +253,52 @@ def test_permissions_endpoint_is_compatible_with_accounts_plugin(self):
         assert uris == ['/buckets/a/collections/b/records/c',
                         '/buckets/a/collections/b',
                         '/buckets/a',
-                        '/accounts/alice']
+                        '/buckets',
+                        '/accounts/alice',
+                        '/accounts']
+
+    def test_account_create_read_write_permissions_(self):
+        resp = self.app.get('/permissions', headers=self.headers)
+        buckets = resp.json['data']
+        accounts = buckets[4]
+        account = buckets[5]
+        self.assertEqual(len(accounts), 5)
+        self.assertEqual(len(account), 3)
+        self.assertEqual(accounts['permissions'], ['read', 'write'])
+        self.assertEqual(accounts['resource_name'], 'account')
+        self.assertEqual(accounts['uri'], '/accounts/alice')
+        self.assertEqual(accounts['account_id'], 'alice')
+        self.assertEqual(accounts['id'], 'alice')
+        self.assertEqual(account['permissions'], ['account:create'])
+        self.assertEqual(account['resource_name'], 'account')
+        self.assertEqual(account['uri'], '/accounts')
```

Comment:
Why `account` singular if it is for `/accounts`?

Reply:
I was following the naming convention for `resource_name`, i.e., `bucket`, `collection`, `group`, and `record`, so thought `account` would be appropriate for the `uri` `/accounts`.

Thanks for the rest of the input too, I'll incorporate them. I'll just wait for your thoughts on `account` vs `accounts` for `resource_name` and then send the changes.

Reply:
ok I get it ! leave it like that sorry :)